pdf_rendering_service/documents/tests/test_services.py

```diff
@@ -0,0 +1,46 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from django.core.files import File
+
+from pdf_rendering_service.documents.models import Document
+from pdf_rendering_service.documents.services.document_store import store_document
+from pdf_rendering_service.documents.services.render import (
+    render_document_into_png_files,
+)
+
+pytestmark = pytest.mark.django_db
+
+
+@patch("pdf_rendering_service.documents.services.document_store.default_storage")
+def test_store_document(mock_default_storage):
+    mock_default_storage.save = Mock()
+    file = File(b"")
+    file.name = "foo"
+    document = store_document(file)
+    assert document.id is not None
+    mock_default_storage.save.assert_called_once()
+    document.delete()
```

Comment:
I quess no, this document does not come from fixture

Reply:
oh, maybe I know what you're thinking. It is not necessary to delete the document because the test runs in transaction which will be rolled back at the end

Reply:
Yes, I was going in that direction 🙂